internal/Peticiones.go

```diff
@@ -0,0 +1,32 @@
+package internal
+
+type Season uint8
+type Semana uint8
+
+const (
+	Invierno  Season = 0
+	Primavera        = 1
+	Verano           = 2
+	Otonio           = 3
+)
+
+const (
+	Lunes Semana = iota //0
+	Martes
+	Miercoles
+	Jueves
+	Viernes
+	Sabado
+	Domingo //6
+)
+
+type Peticiones struct {
+	diasQueQuiero uint
+	temporada     Season
+	dia           Semana
+}
+
+// s.String() devuelve el dia de la semana en string en vez de en numero
+func (s Semana) String() string {
```

Comment:
De veras que llegado a este punto estoy totalmente perdido. Ni entiendo qué relevancia pueden tener los nombres de los días de la semana, ni por qué usáis una función y no una simple estructura de datos, ni nada.